lib/modules/apostrophe-pieces-pages/index.js

```diff
@@ -78,6 +89,7 @@ module.exports = {
 
     self.indexCursor = function(req) {
       return self.pieces.find(req, {})
+        .safeFilters(_.pluck(self.piecesFilters, 'name'))
```

Comment:
FYI - looks like `_.pluck()` was removed from lodash in 4.x: https://github.com/lodash/lodash/wiki/Changelog#v400

Maybe use `_.map()` instead?